dock-schemas.rst

```diff
@@ -0,0 +1,148 @@
+Available schemas
+==================================
+
+Currently, the following schemas are available to use in the Dock protocol:
```

Comment:
I'd rephrase it as such: "Currently dock  provides following schemas as a reference examples"